tokens/utilities/data-json-formatter.js

```diff
@@ -23,11 +23,11 @@ export const format = ({ dictionary, platform, file, options }) => {
 			if (data.ref) {
 				data.ref = convertRef(data.ref);
 
-				if (data.ref === data.value) {
+				if (data.ref?.toString() === data.value?.toString()) {
```

Comment:
This just ensures that the ref is converted to the same type before comparing (sometimes they're numbers).